backend/resources/users.py

```diff
@@ -3,17 +3,33 @@
 from database.models import User
 from flask_restful import Resource
 from helpers.helper_methods import *
-import datetime
+from datetime import *
 import json
 
+
 # TODO: Using the same system, we need to write more apis for all the possible routes
 
 # Login / register route
 
 
 class DefaultPage(Resource):
+    # Return all images in image queue
+    @jwt_required
     def get(self):
-        return 'Welcome to myPanda backend'
+        # Get current requesting user
+        user_id = get_jwt_identity()
+        current_user = User.objects(id=user_id).first()
+        if current_user is None:
+            return {'error': 'Header token is not good, please login again'}, 401
+
+        pictures = current_user.image_queue
+        if len(pictures) == 0:
+            return Response(json.dumps(pictures), mimetype="application/json", status=200)
+
+        pictures = [json.loads(i.to_json()) for i in pictures]
+        pictures.sort(key=lambda x: x['date']['$date']) 
+
+        return Response(json.dumps(pictures), mimetype="application/json", status=200)
```

Comment:
Should this be inside of another route instead of the default page, The default page should simply indicate that the app is running

Reply:
Ah sure, I was thinking of the default page as the user feed, but that can be elsewhere